docker/cpu/Dockerfile

```diff
@@ -32,7 +32,7 @@ WORKDIR $CAFFE_ROOT
 ENV CLONE_TAG=1.0
 
 RUN git clone -b ${CLONE_TAG} --depth 1 https://github.com/chuckcho/video-caffe.git . && \
-    pip2 install --upgrade pip && \
```

Comment:
This may failed with the python3 CI build, I can add the ARG in the dockfile to make the python version fixed.